### ISSUE TITLE
Decide on Requirements for Membership

### ISSUE THREAD

**Issue by asg017** [asg017/os-club-constitution#3]
This completes Article III.

```
Helpful questions:
What dues or fees are required for membership, if any? If dues are required they MUST be included in the constitution.
Who is eligible for membership? 
Are there any restrictions? 
How does one become a member? 
How are members identified? 
Can a member remain in the organization if she/he fails to pay dues?
```

**Comment by asg017** [comment]
My thoughts on this:
> What dues or fees are required for membership, if any? If dues are required they MUST be included in the constitution.

No fees.

> Who is eligible for membership? 

Any UCSD student. (might have to specify undergrads?)

> Are there any restrictions? 

nope

> How does one become a member? 

For this, I think we can either have something like:

1. "Must attend X/Y" of our events/meetings"
2. "Must make a contribution to one of our open source projects"

But 1 might be problematic (what type of events are we even gonna have), and 2 means we have to keep track of those contributions (which shouldn't be too difficult tbh). But we'd also have to decide on what a "contribution" is (a PR, a new issue, a commit?). 

> How are members identified? 

🤷‍♀️  Maybe listed on our website? 

> Can a member remain in the organization if they fails to pay dues?

no fees

**Comment by asg017** [comment]
** whats a contribution **
- Pull Request
- Open an issue?
- Attend a workshop (maybe not this?)
- join a project

**Comment by asg017** [comment]
Closed from #18
